composer/datasets/brats.py

```diff
@@ -257,8 +258,9 @@ def get_data_split(path: str):
         from sklearn.model_selection import KFold
     except ImportError as e:
         raise ImportError(
-            "Composer was installed without unet support. To use unet with Composer, run: `pip install mosaicml[unet]`."
-        ) from e
+            textwrap.dedent("""\
```

Comment:
seems to a Jenkins issue if we need to litter our codebase with `textwrap.dedent`s

Reply:
This isn't a Jenkins issue but rather an effort to not rely on implicit string concatenation...see https://www.python.org/dev/peps/pep-3126 and https://lwn.net/Articles/551426/